src/legacy/core_plugins/metrics/public/components/aggs/percentile.js

```diff
@@ -64,10 +64,12 @@ class PercentilesUi extends Component {
       <EuiFlexItem grow={false}>
         <EuiFieldNumber
           aria-label={intl.formatMessage({ id: 'tsvb.percentile.percentileAriaLabel', defaultMessage: 'Percentile' })}
-          placeholder={intl.formatMessage({ id: 'tsvb.percentile.percentilePlaceholder', defaultMessage: 'Percentile' })}
+          placeholder={0}
```

Comment:
nit: What about this? the append props also add a nice % after the field that can be useful specially for the table viz
```suggestion
          placeholder="0 - 100"
          append={<EuiText size="xs"><strong>%</strong></EuiText>}
```